doc/release/eon_1.0.0.md

```diff
@@ -0,0 +1,58 @@
+# Release version - EON 1.0.0
```

Comment:
let's remove any reference to EON, here we are in the sdk
this release note  is for 0.8.0